### PR TITLE
Update autoconf to 2.71

### PR DESCRIPTION
**Pull request by bgamari**

### CODE REVIEW

**Comment by Bodigrim** [comment]
@bgamari wrong branch?

**Comment by bgamari** [comment]
Yep, looks like it.

**Comment by bgamari** [comment]
Note that I am beginning to second guess myself regarding `config.sub`. I'm still looking for a definitive conclusion on whether it should be committed.

File: configure.ac [hunk 1]
```diff
@@ -1,5 +1,5 @@
-AC_PREREQ([2.60])
-AC_INIT([Haskell unix package], [2.0], [libraries@haskell.org], [unix])
+AC_PREREQ([2.71])
```
Comment (Bodigrim) [inline configure.ac:1]: 2.71 sounds a bit extreme, plenty systems use 2.69 by default. Could it be downgraded?

**Comment by Bodigrim** [comment]
Closing as superseded by #208. If we decide to backport the change to 2.7 branch, I'll update it separately.

**Pull request closed without merging**